src/components/v2/V2Project/ProjectPayerSection.tsx

```diff
@@ -0,0 +1,135 @@
+import { Trans } from '@lingui/macro'
+import { Button, Divider } from 'antd'
+import { TransactorInstance } from 'hooks/Transactor'
+import { useState } from 'react'
+import { TransactionReceipt } from '@ethersproject/providers'
+import FormattedAddress from 'components/shared/FormattedAddress'
+import { JBDiscordLink } from 'components/Landing/QAs'
+
+import { readProvider } from 'constants/readProvider'
+
+const DEPLOY_EVENT_IDX = 0
+
+/**
+ * Return the address of the project payer created from a `deployProjectPayer` transaction.
+ * @param txReceipt receipt of `deployProjectPayer` transaction
+ */
+const getProjectPayerAddressFromReceipt = (
+  txReceipt: TransactionReceipt,
+): string => {
+  const newProjectPayerAddress = txReceipt?.logs[DEPLOY_EVENT_IDX]?.address
+  return newProjectPayerAddress
+}
+
+export default function ProjectPayerSection({
+  useDeployProjectPayerTx,
+}: {
+  useDeployProjectPayerTx: () => TransactorInstance<{}> | undefined
+}) {
+  const [loadingProjectPayer, setLoadingProjectPayer] = useState<boolean>()
+  const [projectPayerAddress, setProjectPayerAddress] = useState<string>()
+  // TODO: load project payer and show different thing in this section if the project already has one
+  // (Issue: #897)
```

Comment:
Thanks for specifying the issue. Totally cool to drop the full link in here too.